src/um/winnt.rs

```diff
@@ -8377,4 +8377,4 @@ pub const ACTIVATION_CONTEXT_SECTION_APPLICATION_SETTINGS: ULONG = 10;
 pub const ACTIVATION_CONTEXT_SECTION_COMPATIBILITY_INFO: ULONG = 11;
 STRUCT!{struct ACTIVATION_CONTEXT {
     dummy: *mut c_void,
-}}
+}}
```

Comment:
Really sorry I'm giving you the total run around here but you still need a newline at the end of the file :P It's all in the CONTRIBUTING.md file